config/default/patch-add-env-var-related-images.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: skupper-controller
+spec:
+  selector:
+    matchLabels:
+      application: skupper-controller
+  template:
+    spec:
+      containers:
+        - name: controller
+          env:
+            - name: RELATED_IMAGE_kube_adaptor
```

Comment:
iiuc, as this patch if for csv bundle, suggest moving the patch to config/manifests...